pipelined/bevy_sprite2/src/bundle.rs

```diff
@@ -29,20 +29,67 @@ impl Default for PipelinedSpriteBundle {
 /// A Bundle of components for drawing a single sprite from a sprite sheet (also referred
 /// to as a `TextureAtlas`)
 #[derive(Bundle, Clone)]
-pub struct PipelinedSpriteSheetBundle {
+pub struct Sprite2dSheetBundle {
```

Comment:
This name reads very awkwardly. Maybe rename the type to SpriteSheet2dBundle instead?

Reply:
Yap, thats better. Same for 3d